prac_08/unreliable_car_test.py

```diff
@@ -0,0 +1,15 @@
+from prac_08.car import Car
+import random
+
+
+class UnreliableCar(Car):
+    def __init__(self, name, fuel, reliability):
+        """Initialise a UnreliableCar instance based on the Car class"""
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def drive(self, distance):
+        if self.reliability < random.randint(0, 100):
```

Comment:
You could add a docstring to your methods.

Reply:
I agree